Server.Core/Request.cs

```diff
@@ -6,15 +6,15 @@ public class Request
     {
         public string HttpVersion { get; internal set; }
         public string Uri { get; internal set; }
-        public string Method { get; internal set; }
+        public HttpMethod Method { get; internal set; }
```

Comment:
Yes, but I am calling them in the builders.  I could change it to where the builders just keep track of the attributes until it actually builds and then call the constructors of either Request or Response.  Is that a better design?